module-gui/gui/widgets/BoxLayout.hpp

```diff
@@ -146,7 +146,7 @@ namespace gui
         VBox(Item *parent, const uint32_t &x = 0, const uint32_t &y = 0, const uint32_t &w = 0, const uint32_t &h = 0);
         virtual ~VBox() = default;
         virtual void addWidget(Item *item) override;
-        auto handleRequestResize(const Item *, Length request_w, Length request_h) -> Size override;
+        virtual auto handleRequestResize(const Item *, Length request_w, Length request_h) -> Size override;
```

Comment:
not needed. tbh, just remove virtual in all cases where `override` is set rather than add virtual` which adds nothing :)